modules/apps/client-extension/client-extension-service/src/main/java/com/liferay/client/extension/internal/service/taglib/BaseDynamicInclude.java

```diff
@@ -112,8 +112,10 @@ public void include(
 			for (CET cet : cets) {
 				GlobalJSCET globalJSCET = (GlobalJSCET)cet;
 
-				if (!StringUtil.equalsIgnoreCase(
-						globalJSCET.getScope(), "instance")) {
+				String scope = globalJSCET.getScope();
+
+				if (!StringUtil.equalsIgnoreCase(scope, "company") &&
+					!StringUtil.equalsIgnoreCase(scope, "instance")) {
```

Comment:
@drewbrokke Here, since we will backport this, we could remove the check for the company, couldn't we?

Reply:
It's ok with me as long as @ethib137 also approves 👍

Reply:
Yeah I think that's okay. Better to move quickly and update this to follow Brian's pattern. @thiago-buarqque we need to make sure that simultaneously our developer docs are updated and the documentation ticket is updated so that we make sure this is communicated correctly.

Reply:
So we will only check for company, not instance.

Reply:
@ethib137 yes.

Reply:
I will speed this up. Thanks guys